nvidia-bert/docker/Dockerfile

```diff
@@ -0,0 +1,53 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+FROM mcr.microsoft.com/azureml/base-gpu:openmpi3.1.2-cuda10.1-cudnn7-ubuntu18.04
```

Comment:
This base image has restrictions to use outside of Azure. See: https://hub.docker.com/_/microsoft-azureml-base-gpu

Will have to build FROM: nvidia/cuda:10.1-cudnn7-devel

Reply:
Will it work like it is if I replace the base image?

Reply:
That does not work:

`/bin/sh: 1: pip: not found`